src/pages/Main/Main.js

```diff
@@ -1,11 +1,110 @@
 import React from 'react';
+import styled from 'styled-components';
+import MainProduct from './MainProduct/MainProduct';
+import MainCarousel from './MainCarousel/MainCarousel';
+import MainShortCutItem from './MainShortCutItem/MainShortCutItem';
+
+const MAIN_SHORTCUT = [
+  { id: 1, img: 'shortcut_img_url', categoryName: '럭키박스' },
+  { id: 2, img: 'shortcut_img_url', categoryName: '남성추천' },
+  { id: 3, img: 'shortcut_img_url', categoryName: '여성추천' },
+  { id: 4, img: 'shortcut_img_url', categoryName: '셀럽픽' },
+  { id: 5, img: 'shortcut_img_url', categoryName: '이번주 브랜드관' },
+  { id: 6, img: 'shortcut_img_url', categoryName: '정가 아래' },
+  { id: 7, img: 'shortcut_img_url', categoryName: '인기 럭셔리' },
+  { id: 8, img: 'shortcut_img_url', categoryName: '연말 선물' },
+  { id: 9, img: 'shortcut_img_url', categoryName: '겨울 코디' },
+  { id: 10, img: 'shortcut_img_url', categoryName: '수수료 할인' },
+];
 
 const Main = () => {
   return (
-    <div>
-      <div>Main Page</div>
-    </div>
+    <MainWrapper>
+      <MainCarousel />
+      <MainBox>
+        <MainShortCutList>
+          {MAIN_SHORTCUT.map(({ id, categoryName }) => {
+            return <MainShortCutItem key={id} title={categoryName} />;
+          })}
+        </MainShortCutList>
+
+        <MainBanner>
+          <button>지금 거래하기</button>
+        </MainBanner>
+        <MainProductAlignBox>
+          <MainCategoryTitle>Just Dropped</MainCategoryTitle>
+          <MainCategorySubTitle>발매 상품</MainCategorySubTitle>
+          <MainProduct />
+          <MainProduct />
+        </MainProductAlignBox>
+        <MainBanner>
+          <button>지금 거래하기</button>
+        </MainBanner>
+        <MainProductAlignBox>
+          <MainCategoryTitle>Just Dropped</MainCategoryTitle>
+          <MainCategorySubTitle>발매 상품</MainCategorySubTitle>
+          <MainProduct />
+          <MainProduct />
+        </MainProductAlignBox>
+      </MainBox>
+    </MainWrapper>
   );
 };
 
 export default Main;
+
+const MainWrapper = styled.div`
+  width: 100%;
+`;
+
+const MainBox = styled.div`
+  margin-top: 35px;
+`;
+
+const MainShortCutList = styled.div`
+  width: 80%;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const MainBanner = styled.div`
+  width: 100%;
+  height: 400px;
+  margin-top: 45px;
+  background-color: ${props => props.theme.mainBrandBlack};
```

Comment:
일반적으로 theme prop을 사용할 때는 바로 구조분해 할당을 해주는게 보편적입니다.

```suggestion
  background-color: ${({ theme })=> theme.mainBrandBlack};
```

Reply:
넵 수정완료했습니다 ! 감사합니다 :)